integration/clean/clean_dedicated_host.go

```diff
@@ -0,0 +1,70 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build clean
+// +build clean
+
+package main
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"log"
+	"time"
+)
+
+// Can't release a host if it was being used within the last 24 hr add 2 hr as a buffer
+var expirationDateDedicatedHost = time.Now().UTC().Add(keepDurationDedicatedHost)
+
+const keepDurationDedicatedHost = -1 * time.Hour * 26
+
+func cleanDedicatedHost() []error {
```

Comment:
After looking through the function, there's no point at which we would ever return an errors array that has > 1 element in it. Can you change this to 
```suggestion
func cleanDedicatedHost() error {
```
and just return a single err